LunaDll/GameConfig/GameAutostart.h

```diff
@@ -29,6 +20,14 @@ class GameAutostart
     void setSaveSlot(int val) { saveSlot = val; }
 
     bool applyAutostart();
+    
+    //Settings
```

Comment:
Do these really need to be public?

Reply:
The reason why they are public is because it's used for the LaunchEpisode function when booting SMBX2. I could move the LaunchEpisode function though to GameAutostart, if that'll be better